scripts/dotnet-cli-build/PrepareTargets.cs

```diff
@@ -200,6 +181,172 @@ public static BuildTargetResult RestorePackages(BuildTargetContext c)
             return c.Success();
         }
 
+        [Target]
+        [BuildPlatforms(BuildPlatform.Ubuntu)]
+        public static BuildTargetResult CheckUbuntuDebianPackageBuildDependencies(BuildTargetContext c)
+        {
+            var debianPackageBuildDependencies = new string[]
+            {
+                "devscripts", 
+                "debhelper",
+                "build-essential"
+            };
+
+            var messageBuilder = new StringBuilder();
+
+            foreach (var package in debianPackageBuildDependencies)
+            {
+                if (!AptPackageIsInstalled(package))
+                {
+                    messageBuilder.Append($"Error: Debian package build dependency {package} missing.");
+                    messageBuilder.Append(Environment.NewLine);
+                    messageBuilder.Append($"-> install with apt-get install {package}");
+                    messageBuilder.Append(Environment.NewLine);
+                }
+            }
+
+            if (messageBuilder.Length == 0)
+            {
+                return c.Success();
+            }
+            else
+            {
+                return c.Failed(messageBuilder.ToString());
+            }
+        }
+
+        [Target]
+        [BuildPlatforms(BuildPlatform.Ubuntu)]
+        public static BuildTargetResult CheckUbuntuCoreclrDependencies(BuildTargetContext c)
+        {
+            var errorMessageBuilder = new StringBuilder();
+
+            var ubuntuCoreclrDependencies = new string[]
+            {
+                "unzip",
+                "curl",
+                "libicu-dev",
+                "libunwind8",
+                "gettext",
+                "libssl-dev",
+                "libcurl3-gnutls",
+                "zlib1g",
+                "liblttng-ust-dev",
+                "lldb-3.6-dev",
```

Comment:
These are intended to be the runtime dependencies of coreclr which we need to run the build of the CLI.

IIRC lldb-3.6-dev was required to satisfy the exports for one of the .so files, the debian package wouldn't build without it.